$NetBSD: patch-Makefile,v 1.12 2022/08/19 08:59:48 fcambus Exp $

- Do not hard-code -pthread, as it is handled by the pkgsrc infrastructure.
- Do not hard-code -ldl

--- Makefile.orig	2022-08-18 08:27:51.000000000 +0000
+++ Makefile
@@ -49,7 +49,7 @@ MOLD_CXXFLAGS := -std=c++20 -fno-excepti
                  -fno-asynchronous-unwind-tables \
                  -DMOLD_VERSION=\"$(VERSION)\" -DLIBDIR="\"$(LIBDIR)\""
 
-MOLD_LDFLAGS := -pthread -lz -lm -ldl
+MOLD_LDFLAGS := -lz -lm
 
 LTO = 0
 ifeq ($(LTO), 1)
